notes/running-mobius-app.md

```diff
@@ -173,9 +173,9 @@ Displays the number of XML elements in the input XML file provided as the first
 
 ### Hive Example (Sql)
 * 
-`sparkclr-submit.cmd --jars <jar files used for using Hive in Spark> --exe HiveDataFrame.exe C:\Git\Mobius\examples\Sql\HiveDataFrame\bin\Debug`
+`sparkclr-submit.cmd --jars <jar files used for using Hive in Spark> --exe HiveDataFrame.exe C:\Git\Mobius\examples\Sql\HiveDataFrame\bin\Debug <input json file path>`
```

Comment:
Actually, I did not use `--jars` to include any jars files used for using Hive in Spark since the pre-built Spark-1.6.2 assembly package already contain these hive related class. I think I'll remove this part in description.